pages/PhentermineForm.tsx

```diff
@@ -0,0 +1,225 @@
+    'I will NOT take any other diet medication (over the counter or prescribed) while on Phentermine.',
+    'I agree to return to the office for evaluation of any side effects or problems associated with Phentermine use.',
+    'I will not drive or operate machinery till I know how Phentermine will affect me.',
+    'I understand that lost/stolen prescriptions will not be refilled early, nor will Phentermine be prior authorized if not covered by my insurance plan.',
+    'I will not stop Phentermine suddenly without talking to my health care provider.',
+    'I have received and reviewed PHENTERMINE FACTS sheet.',
+  ]
+
+  // Handles changes in the initials fields
+  const handleInitialChange = (index: number, value: string) => {
+    const newInitials = [...initials]
+    newInitials[index] = value
+    setInitials(newInitials)
+  }
+
+  // Handles form submission
+  const handleSubmit = () => {
+    // Format date of birth for submission
+    const formattedDob = `${dateOfBirth.month}-${dateOfBirth.day}-${dateOfBirth.year}`
+
+    // Validation to ensure all fields are filled
+    if (
+      !patientName ||
+      dateOfBirth.day === '' ||
+      dateOfBirth.month === '' ||
+      dateOfBirth.year === '' ||
+      !patientSignature ||
+      !witnessSignature ||
+      !date ||
+      initials.some((initial) => initial === '')
+    ) {
+      toast.error('Please fill out all fields.')
+      return
+    }
+
+    // Constructing the form data for submission
+    const formData = {
+      patientId: patientName.replace(/\s+/g, '-').toLowerCase(),
+      patientName,
+      dateOfBirth: formattedDob,
+      initials,
+      patientSignature,
+      witnessSignature,
+      date,
+    }
+
+    // Submit the form data to the server/database
+    submitPhentermineForm(formData)
+      .then(() => {
+        toast.success('Form submitted successfully!')
+
+        // Reset form fields to initial state
+        setPatientName('')
+        setDateOfBirth({ month: '', day: '', year: '' })
+        setInitials(new Array(12).fill(''))
+        setPatientSignature('')
+        setWitnessSignature('')
+        setDate('')
+
+        // Delay the sign-out and redirection by 2 seconds
+        setTimeout(async () => {
+          auth.signOut() // Sign out the user
+          router.push('/SubstanceContractThankYou') // Redirect to Thank You page
+        }, 1000)
+      })
+      .catch((error) => {
+        toast.error('Error submitting form: ' + error.message)
+      })
+  }
+
+  return (
+    <div>
+      <ToastContainer position={'bottom-right'} />
+      <Header selectCompany={'AMA'} routePatientsHome={true} />
+      <div className="max-w-4xl mx-auto bg-white shadow-md rounded-lg p-6 mt-8">
+        <h1 className="text-center text-3xl font-bold text-gray-700 mb-6">
+          Phentermine Contract
+        </h1>
+        <div className="space-y-4">
+          <div className="flex justify-between items-center">
+            <input
```

Comment:
it appears you are using the standard react input. you should be using a react component instead of repeating styled code. I know i created one in this project try to use components as much ass possible. if you need to make a change to all of the inputs it'l be hell.